frontend/src/components/QuestionAndAnswer/QuestionAndAnswerForm.tsx

```diff
@@ -32,12 +43,29 @@ const QuestionAndAnswerForm = ({ questionNumber, question, answer, disabled, onA
         }
     }, [localAnswer])
 
+    useEffect(() => {
+        if (isSavingAnswer) {
+            setSavingState(SavingState.Saving)
+        } else {
+            if (savingState === SavingState.Saving) {
+                setSavingState(SavingState.Saved)
+            } else {
+                setSavingState(SavingState.None)
+            }
+        }
+    }, [isSavingAnswer])
+
     return (
         <>
             <Grid container>
                 <Grid item xs={12}>
                     <Box display="flex" flexDirection="row-reverse">
-                        <Chip primary title={organizationToString(question.organization)} />
+                        <Box display="flex" justifyContent="center" m={1}>
+                            <Chip primary title={organizationToString(question.organization)} />
+                        </Box>
+                        <Box display="flex" justifyContent="center" m={1}>
+                            <SaveIndicator savingState={savingState} />
+                        </Box>
```

Comment:
Saving a little margin by only specifying the right-margin and save some code by moving the centering up. 
```suggestion
                        <Box display="flex" justifyContent="flex-end" alignItems="center">
                        <Box mr={1}>
                            <SaveIndicator savingState={savingState} />
                        </Box>
                        <Chip primary title={organizationToString(question.organization)} />
                    </Box>
```

Reply:
Agreed, but increasing the right-margin then, because I think it was a little too close with only 8px